airflow/api_fastapi/core_api/serializers/assets.py

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
This should now move to `airflow/api_fastapi/core_api/datamodels/assets.py` after we merged PR: https://github.com/apache/airflow/pull/43823